api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -274,6 +284,23 @@ message UIDiscoverDeployServiceEvent {
   DeployType deploy_type = 5;
 }
 
+// UIDiscoverCreateDiscoveryConfigEvent is emitted when a discovery config is successfully created.
+message UIDiscoverCreateDiscoveryConfigEvent {
```

Comment:
we should also probably be emitting `skipped` event for like `enroll rds | eks | ec2` when auto discovery is on and also emit discovery config event

Reply:
Agreed, we should start emitting this event for other resources